### PR TITLE
T73556 - Login to BRO

### PR DESCRIPTION
**Pull request by ArielWRojas**

### CODE REVIEW

File: IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿namespace IDT.XGen.Main.UI.Pages.BRO
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/Login/ILoginPage.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace IDT.XGen.Main.UI.Pages.Login
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/Login/ILoginPage.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿namespace IDT.XGen.Main.UI.Pages.Login
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+{
+    using System.ComponentModel;
+    using IDT.XGen.Main.UI.Pages;
+    using IDT.XGen.Main.UI.Pages.Distributor;
+
+    public class LoginFactory
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/PageTransporter.cs [hunk 5]
```diff
@@ -0,0 +1,78 @@
+﻿namespace IDT.XGen.Main.UI.Pages
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/PageTransporter.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/PortalWeb.cs [hunk 6]
```diff
@@ -0,0 +1,8 @@
+﻿namespace IDT.XGen.Main.UI.Pages
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/PortalWeb.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/PortalWeb.cs [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+{
+    public enum PortalWeb
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/PortalWeb.cs]: Add documentation

File: IDT.XGen/Tests/UI/StepDefinitions/LoginSteps.cs [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    using IDT.XGen.Main.UI.Pages;
+    using IDT.XGen.Main.UI.Pages.Login;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class LoginSteps
+    {
+        [Given(@"I login to ""(.*)"" portal web with ""(.*)"" credentials")]
+        [When(@"I login to ""(.*)"" portal web with ""(.*)"" credentials")]
+        public void GivenILoginToWebWithCredentials(string portalWeb, string userType)
```
Comment (mabalderramav) [inline IDT.XGen/Tests/UI/StepDefinitions/LoginSteps.cs]: change When and Give for StepDefinition.
For Example:
`[StepDefinition(@"I login to ""(.*)"" portal web with ""(.*)"" credentials")]`

File: IDT.XGen/Main/UI/Pages/UserType.cs [hunk 9]
```diff
@@ -0,0 +1,8 @@
+﻿namespace IDT.XGen.Main.UI.Pages
+{
+    public enum UserType
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/UserType.cs]: Add documentation

File: IDT.XGen/Main/UI/Pages/UserType.cs [hunk 10]
```diff
@@ -0,0 +1,8 @@
+﻿namespace IDT.XGen.Main.UI.Pages
```
Comment (mabalderramav) [inline IDT.XGen/Main/UI/Pages/UserType.cs]: add documentation

File: IDT.XGen/Main/App/Environment.cs [hunk 11]
```diff
@@ -0,0 +1,52 @@
+﻿namespace IDT.XGen.Main.App
+{
+
```
Comment (mabalderramav) [inline IDT.XGen/Main/App/Environment.cs]: Remove this blank line.

File: IDT.XGen/Main/App/Environment.cs [hunk 12]
```diff
@@ -0,0 +1,52 @@
+﻿namespace IDT.XGen.Main.App
```
[the remark this answers is not in the document]
Comment (ArielWRojas) [reply IDT.XGen/Main/App/Environment.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs (hunk 1 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+{
+
```
[the remark this answers is not in the document]
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs]: Ok, I removed it

File: IDT.XGen/Main/UI/Pages/Login/ILoginPage.cs (hunk 2 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/Login/ILoginPage.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs (hunk 3 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs (hunk 4 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/Login/LoginFactory.cs]: Ok, I added the following documentation: 
This class is to hangle the login pages of the different portals.

File: IDT.XGen/Main/UI/Pages/PageTransporter.cs (hunk 5 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/PageTransporter.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/PortalWeb.cs (hunk 6 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/PortalWeb.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/PortalWeb.cs (hunk 7 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/PortalWeb.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/UserType.cs (hunk 10 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/UserType.cs]: Ok, I added it

File: IDT.XGen/Main/UI/Pages/UserType.cs (hunk 9 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/UserType.cs]: Ok, I added it

File: IDT.XGen/Tests/UI/StepDefinitions/LoginSteps.cs (hunk 8 above)
Comment (ArielWRojas) [reply IDT.XGen/Tests/UI/StepDefinitions/LoginSteps.cs]: Ok, I updated the step definition

**Pull request merged**